CONTRIBUTING.md

```diff
@@ -53,7 +53,22 @@ about Git.
   - Create pull request
 
 The title of the pull request should be prefixed by the component or area that
-the pull request affects. Examples:
+the pull request affects. Valid areas as:
+
+  - *Consensus* for changes to consensus critical code
```

Comment:
I tend to always use `<component>: <title>` as in commit messages of the Linux kernel and many related projects.